agile-development/team-agreements/team-manifesto/readme.md

```diff
@@ -0,0 +1,78 @@
+# Team Manifesto
+
+## Introduction
+
+CSE teams work with a new development team in each customer engagement which requires a phase of introduction & knowledge transfer before starting an engagement.
+
+Completion of this phase of ice-breakers and discussions around the standards takes time however also required to start increasing the learning curve of the new team.
+
+A team manifesto is a soft contract among team members which summarizes the basic principles and values of the team and aiming to provide a consensus about
+expectations from each team member in order to deliver high quality output at the end of each engagement.
+
+It aims to reduce the time on setting the right expectations and providing a consensus among team members as well as clarifying the question of "How does the new team develop the software?"
+
+Another main goal of the manifesto is providing a discussion environment during "manifesto building session" and detect the gaps/differences in certain topics for each member during these discussions.
+
+It also serves in the same way when a new team member joins to the team. It gives a short summary of standards and if the new team member asks any question about any topic, this is a good thing to understand the improvement area and take the required actions.
+
+## How to Build a Team Manifesto
+
+It can be said that the best time to start building it is at the very early phase of the engagement when teams meet with each other for swarming or during the preparation phase.
+
+It is recommended to keep team manifesto as simple as possible, so preferably, one-page simple document which **doesn't include any references or links** is a nice format for it.
+If there is a need for providing knowledge on certain topics, the way to do is delivering brown-bag sessions, technical katas, practices documentations and others later on.
+
+A few important points about team manifesto;
+
+- Team manifesto is built by the **team itself**
+
+- It can include any technical or behavioural agility topics as soon as team find it relevant,
+
+- It aims to have a common understanding about the desired expertise, practices and/or behaviours within the team
+
+- Based on the needs of the team and retrospective results, it can be modified during the engagement.
+
+In Microsoft, we have certain standards to build high quality and well-crafted software as well as to give comfortable/transparent environment to each team member for helping her/him to reach her/his highest potential.
+
+The difference of team manifesto from other comprehensive set of standards/practices documents is, it is used to give a short summary of the upcoming way of working and practices to prepare a base before deep dive starts with open discussion environment in an ice-breaker way.
```

Comment:
```suggestion
The difference between the team manifesto and other team documents is that it is used to give a short summary of expectations of how we should behave and work in the team, before we dive into the technical work.
```

Reply:
just changed the end of the sentence with "code with sprint" instead of technical work but mostly applied in commit 493c61df34e91f74e503658642e0a329692b61f2